druid/src/contexts.rs

```diff
@@ -708,15 +680,15 @@ impl<'a> UpdateCtx<'a> {
     }
 }
 
-impl<'a, 'b> LayoutCtx<'a, 'b> {
+impl<'a, 'b, 'c> LayoutCtx<'a, 'b, 'c> {
```

Comment:
Might be useful for the macro introduced by #972 :
This `'b` and `RootState`s  `'a` are the only required lifetimes in this file (for two functions only), all others could be elided using `'_`.

Reply:
interesting, I'll play around with that.

Reply:
cool, I was able to simplify the lifetimes quite a bit, i'd forgotten they'd relaxed that rule :)

Reply:
I also still keep adding them and then only realize afterwards that it was completely unnecessary :sweat_smile: